Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -260,10 +260,14 @@ - (RLMSyncUser *)createAdminUserForURL:(NSURL *)url username:(NSString *)usernam
 }
 
 + (NSString *)retrieveAdminToken {
+#if PROVIDING_OWN_ROS
+    return @"eyJhcHBfaWQiOiJpby5yZWFsbS5hdXRoIiwiaWRlbnRpdHkiOiJfX2FkbWluIiwiYWNjZXNzIjpbImRvd25sb2FkIiwidXBsb2FkIiwibWFuYWdlIl0sInNhbHQiOiI0YTM0NmVlNyJ9:WmB4swR8pQoWfLn1z06z9icasvIBxtLAMcsmt9AfC2bBMWN/4jx/MyuxZVnaassjikvMLp/yfFc4OBPhrpJsm0rjaAtTODaq242qQCaY/eFy5SOd5BH+nJ9WPsW6ByN/ZpccqlaY6ihboEdxCZZRTlIUwy92oH9ljeUzDW1JuURjPnYPmGdAUDnGp+aB9ZHtC/SrKva2J8IGrI2npPZxq0EmN9cbjvo/NQQlBwYqmTX3K0Fa3UmvvxGAuySVpx2ynGUrNsD6LqNkMlO4rkPme49BKJsG40y69itExj0gx45FGYVCd+USba1um4kLusEcePMxR6/Q2kR504M+C68PWA==";
```

Comment:
This has to be manually set to the token from your ROS instance, right? It'd be good to have all the things that need to be touched for that in one place. Could have this come from a define set in the same place as `PROVIDING_OWN_ROS` (or maybe even combine the two).

Reply:
That's a good idea.